docs/acknowledgements.html

```diff
@@ -1,28 +1,137 @@
-<h1>Acknowledgements</h1>
+<h1 class="title-1">Acknowledgements</h1>
 
-<p>The core contributors of the team so far are:
-<ul>
-  <li><a href="https://github.com/peterebden">Peter Ebden</a></li>
-  <li><a href="https://github.com/csdigi">Will Montgomery</a></li>
-  <li><a href="https://github.com/FS89">Fabian Siddiqi</a></li>
-  <li><a href="https://github.com/dianacostea">Diana-Maria Costea</a></li>
-  <li><a href="https://github.com/dimpavloff">Dimitar Pavlov</a></li>
-  <li><a href="https://github.com/williambhot">William Bhot</a></li>
-  <li><a href="https://github.com/agenticarus">Mike Cripps</a></li>
-  <li><a href="https://github.com/BNMetrics">Luna Chen</a></li>
-  <li><a href="https://github.com/tatskaari">Jon Poole</a></li>
+<p>The core contributors of the team so far are:</p>
+
+<ul class="bulleted-list">
+  <li>
+    <span>
+      <a
+        class="copy-link"
+        href="https://github.com/peterebden"
+        target="_blank"
+        rel="noopener"
+        >Peter Ebden</a
+      >
```

Comment:
Is this standard formatting?

Reply:
Prettier (https://prettier.io/) is opinionated on how it formats source code. It looks weird at first, I must say

Reply:
Also, `<a>Text</a>` is different than `<a> Text </a>` (or new line instead of spaces between the text).